forc-plugins/forc-client/test/data/big_contract/src/main.sw

```diff
@@ -0,0 +1,14 @@
+contract;
+
+abi MyContract {
+    fn test_function() -> bool;
+}
+
+impl MyContract for Contract {
+    fn test_function() -> bool {
```

Comment:
It would be good to add more methods to make this a more complex ABI. It's worth having a test that calls different methods in the same chunked contract.

Reply:
Resolving this as we no longer use abi to generate the loader, we rely on sdk to create a loader with asm directly.

Reply:
I think this is important. I'm not sure I understand why we wouldn't want this test.